Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ansi-to-tui"
-version = "7.0.1"
+version = "8.0.0-alpha.0"
```

Comment:
Let's call it -beta.0 to align with ratatui's version, also this probably breaks release-plz automation if we bump it here. We probably should not bump this, and bump the version in the release PR instead (explicitly changing that to -beta.0)

Reply:
Yup